modules/logging-loki-storage-aws.adoc

```diff
@@ -25,3 +25,23 @@ $ oc create secret generic logging-loki-aws \
   --from-literal=access_key_secret="<aws_access_key_secret>" \
   --from-literal=region="<aws_region_of_your_bucket>"
 ----
+
+== AWS storage for STS enabled clusters
+
+If your cluster has STS enabled, the Cloud Credential Operator (CCO) supports short-term authentication using AWS tokens.
+
+.Required data
+bucketnames:: Comma-separated list of bucket names.
```

Comment:
Same for the secret by CCO on AWS, I suggest we also specify the outline/sample of the secret created by the CCO and the description of those attributes (which are mentioned here)